src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,7 @@
+$text-color-first: #060b35;
```

Comment:
it is a good practice to name variables with color values with `$color` name at the start, e.g.

```
$color-first-text
$color-blue
$color-white
$color-second-text
```

etc. please rename them to following convention :)